<!doctype html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <link rel="stylesheet" href="css/reveal.css">
    <link rel="stylesheet" href="css/theme/simple.css" id="theme">
    <link rel="stylesheet" href="lib/css/zenburn.css">
    <link rel="stylesheet" href="plugin/toc-progress/toc-progress.css">
    <link rel="stylesheet" href="plugin/title-footer/title-footer.css">
    <link rel="stylesheet" href="plugin/menu/menu.css">
    <style>
        hr { border: 1px solid darkred; }
        .reveal h1 { font-size: 160%;text-transform:none }
        .reveal h2 { font-size: 140%;text-transform:none;text-align:left;color:#900 }
        .reveal h3 { font-size: 120%;text-transform:none }
        .reveal h4 { font-size: 105%;text-transform:none }
        .reveal h5 { font-size: 90% }
        .reveal h6 { font-size: 80% }
        .reveal p, .reveal ol li, .reveal ul li { font-size: 72%;text-align:left }
    </style>

    <title>reveal.js template</title>
</head>

<body>
    <div class="reveal">
        <div class="slides">
            <section>
                <section>
                    <h1>Integrated Omics, Machine Learning,<br/> and Data Mining</h1>
          <hr />
                    <div style="display:grid;gap:15px">
            <img style="grid-row:1;grid-column:1;width:66%;margin:auto" src="resource/SJTU.png" />
            <img style="grid-row:1;grid-column:2;width:50%;margin:auto" src="resource/LSU.png" />
            <img style="grid-row:1;grid-column:3;width:66%;margin:auto" src="resource/CU.png" />
            <img style="grid-row:2;grid-column:1;width:66%;margin:auto" src="resource/Asuragen.png" />
            <img style="grid-row:2;grid-column:2;width:75%;margin:auto" src="resource/Myokardia.png" />
            <img style="grid-row:2;grid-column:3;width:92%;margin:auto" src="resource/BMS.png" />
            <p style="grid-row:1/3;grid-column:4;width:95%;margin:auto;text-align:right">
              <br /><br /><br /><br />
              Rui Xiao, Ph.D.<br />
              Principal Scientist<br />
              Informatics & Predictive Sciences<br />
              Bristol Myers Squibb<br />
                            2022-08-05
                        </p>
                    </div>
                </section>
                <section>
                    <h2>Selected Publications</h2>
                    <ol style="font-size:20px">
            <li><b>Xiao R</b>, Goldklang MP, Stearns KN, D'Armiento JM. Deep Learning in Computer Vision Provides
              Valuable Insights and Improved Efficiency for Lung Cytology and Histopathology. Am J Respir Crit Care Med.
              2019;199:A5761</li>
            <li>Goldklang MP, Tekabe Y, Zelonina T, Trischler J, <b>Xiao R</b>, Stearns K, Rodriguez K, Shields A,
              Romanov A, D'Armiento JM, Johnson LL. Single-photon emission computed tomography/computed tomography
              imaging of RAGE in smoking-induced lung injury. Respir Res. 2019 Jun 10;20(1):116.</li>
            <li>Dunham-Snary KJ, Sandel MW, Sammy MJ, Westbrook DG, <b>Xiao R</b>, McMonigle RJ, Ratcliffe WF, Penn A,
              Young ME, Ballinger SW. Mitochondrial - nuclear genetic interaction modulates whole body metabolism,
              adiposity and gene expression in vivo. EBioMedicine. 2018 Oct;36:316-328</li>
            <li><b>Xiao R</b>, Goldklang MP, D'Armiento JM. Reply: Parenchymal Airspace Profiling Has Its Merits and Is
              a Valuable Addition to Existing Methods for Lung Morphometry. Am J Respir Cell Mol Biol. 2017
              Jul;57(1):132-133.</li>
            <li>Noël A, <b>Xiao R</b>, Perveen Z, Zaman H, Le Donne V, Penn A. Sex-specific lung functional changes in
              adult mice exposed only to second-hand smoke in utero. Respir Res. 2017 Jun 27;18(1):104.</li>
            <li>Goldklang MP, Tekabe Y, Zelonina T, Trischler J, <b>Xiao R</b>, Stearns K, Romanov A, Muzio V, Shiomi T,
              Johnson LL, D'Armiento JM. Single-Photon Emission Computed Tomography/Computed Tomography Imaging in a
              Rabbit Model of Emphysema Reveals Ongoing Apoptosis In Vivo. Am J Respir Cell Mol Biol. 2016
              Dec;55(6):848-857.</li>
            <li><b>Xiao R</b>, Goldklang MP, D'Armiento JM. Parenchymal Airspace Profiling: Sensitive Quantification and
              Characterization of Lung Structure Evaluating Parenchymal Destruction. Am J Respir Cell Mol Biol. 2016
              Nov;55(5):708-715.</li>
            <li>Xie T, Liang J, Liu N, Huan C, Zhang Y, Liu W, Kumar M, <b>Xiao R</b>, D'Armiento J, Metzger D, Chambon
              P, Papaioannou VE, Stripp BR, Jiang D, Noble PW. Transcription factor TBX4 regulates myofibroblast
              accumulation and lung fibrosis. J Clin Invest. 2016 Aug 1;126(8):3063-79.</li>
            <li><b>Xiao R</b>, Noël A, Perveen Z, Penn AL. In utero exposure to second-hand smoke activates
              pro-asthmatic and oncogenic miRNAs in adult asthmatic mice. Environ Mol Mutagen. 2016 Apr;57(3):190-9.
            </li>
            <li>Noël A, <b>Xiao R</b>, Perveen Z, Zaman HM, Rouse RL, Paulsen DB, Penn AL. Incomplete lung recovery
              following sub-acute inhalation of combustion-derived ultrafine particles in mice. Part Fibre Toxicol. 2016
              Feb 24;13:10.</li>
            <li><b>Xiao R</b>, Perveen Z, Rouse RL, Le Donne V, Paulsen DB, Ambalavanan N, Penn AL. In utero exposure to
              second-hand smoke aggravates the response to ovalbumin in adult mice. Am J Respir Cell Mol Biol. 2013
              Dec;49(6):1102-9.</li>
            <li>Nelson P, Ngoc Tran TD, Zhang H, Zolochevska O, Figueiredo M, Feng JM, Gutierrez DL, <b>Xiao R</b>, Yao
              S, Penn A, Yang LJ, Cheng H. Transient receptor potential melastatin 4 channel controls calcium signals
              and dental follicle stem cell differentiation. Stem Cells. 2013 Jan;31(1):167-77.</li>
            <li><b>Xiao R</b>, Perveen Z, Paulsen D, Rouse R, Ambalavanan N, Kearney M, Penn AL. In utero exposure to
              second-hand smoke aggravates adult responses to irritants: adult second-hand smoke. Am J Respir Cell Mol
              Biol. 2012 Dec;47(6):843-51.</li>
            <li>Bui A, <b>Xiao R</b>, Perveen Z, Kleinow K, Penn A. Zebrafish embryos sequester and retain petrochemical
              combustion products: developmental and transcriptome consequences. Aquat Toxicol. 2012 Feb;108:23-32.</li>
                    </ol>
                </section>
            </section>

            <section>
                <section>
                    <h1>LSU-PhD</h1>
          <hr style="margin:5%" />
          <h4>Physiological responses, transcriptomic, epigenomic consequences on adult mice exposed in utero to smoke exposure -
            Inflammation, Asthma, ...</h4>
                    <img style="max-width:24vw;max-height:18vh" src="resource/LSU.png" />
                </section>
                <section>
                    <h2>Smoking prevalence and detrimental effects</h2>
          <hr />
                    <div style="display:grid;gap:15px;">
                    <div style="grid-column:1/4">
              <ul style="float:left">
                        <li>According to <i>"Pregnancy Risk Assessment and Monitoring System, 2010."</i> </li>
                            <ul>
                            <li><b>25%</b> of adults smoke regularly; </li>
                            <li><b>10.7%</b> females smoke during last 3 months of pregnancy.</li>
                            </ul>
                <li style="padding-top:8px">Smoke exposure is a major <b>risk factor</b> for <b>chronic lower
                    respiratory disease</b> and <b>cancer</b>, both of which are <b>leading causes of death</b> in the
                  US.</li>
                <li style="padding-top:8px"><b>Epidemiological studies</b> have associated in utero or prenatal exposure
                  to second-hand smoke (SHS) with <b>altered lung function</b> and <b>exacerbated asthmatic
                    responses</b> in children.</li>
                        </ul>
                    </div>
                    <img style="grid-column:4" src="resource/SmokeMother.jpg" />
                    </div>
                </section>
                <section>
                    <h2>Animal models to study the effects of in utero smoke exposure</h2>
          <div style="display:grid;gap:15px" class="#fragment">
            <p style="grid-row:1;grid-column:1/4;float:left">
              <b>Animal experiments</b> can be designed to: i) expose pregnant female mice to side-stream smoke
                during day 6-19 of <b>gestation</b>; ii) assess the postnatal or adult <b>responses to environmental
                  stressors</b>.
              <img style="margin:0% 7%;max-width:85%;" src="resource/SHSOVATimeline.png" />
            </p>
            <img style="grid-row:1;grid-column:4" src="resource/SmokeMouse.jpg" />
          </div>
          <p style="margin:5px">
            Adult Allergen, Ovalbumin, (<b>A</b>IR/<b>S</b>HS, <b>O</b>VA, <b>F</b>emale/<b>M</b>ale)
                          <ul>
                            <li>Physiological Responses (airway hyper-responsiveness, breathing frequency)</li>
                            <li>Transcriptome (microarray, mRNA-seq)</li>
                            <li>Epigenome (miRNA-seq)</li>
                          </ul>
          </p>
                </section>
                <section>
                    <h2>SHS-OVA groups exhibited increased airway hyper-responsiveness and decreased breathing frequency</h2>
                    <p>
            † (SOM vs SOF, sex difference) @ 3,6,12 for airway hyper-responsiveness (AHR). <br />
            ‡ (SO vs AO, female and male pooled) @ 6,50 for AHR, @ 25 for breathing frequency (f). <br />
                    </p>
                    <img style="height:40vh" src="resource/AS_O_FM_AHR.png" />
                </section>

                <section>
                    <h2>Comparison between microarray and RNA-seq for transcriptome profiling</h2>
          <p>Advantages of RNA sequencing , compared to microarray:</li></p>
                    <ul>
                        <li>Increased sensitivity: actual counts vs fluorescent intensity.</li>
            <li>Unbiased detection: no pre-designed probes needed and suitable for RNAs of all sizes, including
              messengerRNAs (mRNA) and microRNAs (miRNA).</li>
                    </ul>
          <div style="display:grid;gap:15px">
            <img style="grid-column:1;height:22vh;margin:auto" src="resource/Microarray.png" />
            <img style="grid-column:2;height:12vh;margin:auto" src="resource/RNAseq.png" />
                    </div>
                </section>

                <section>
                    <h2>Our findings also largely confirmed the increased sensitivity of RNA-seq</h2>
          <p>Across pairwise comparisons with different baseline expression levels:</p>
                    <ul>
                        <li>Medium-high correlation for top 25% <b>high</b> abundance genes: orange dots</li>
                        <li>Medium-low correlation for <b>middle</b> 50% : green dots</li>
                        <li>Very-low correlation for bottom 25% <b>low</b> abundance genes: blue dots</li>
                    </ul>
                    <img style="height:38vh" src="resource/RNAseq-Microarray.png" />
                </section>

        <section>
          <h2>Differentially expressed genes were identified for SOF vs AOF and SOM vs AOM.</h2>
          <p>
            Biplot of both pairwise comparisons, labels: FDR&lt;0.05. <br/>
            There are more up-regulated genes and more differentially expressed between males.
          </p>
          <div style="display:grid;gap:15px;">
          <img style="grid-column:1" src="resource/SOFM_biplot1a.png" />
          <img style="grid-column:2" src="resource/SOFM_biplot2.png" />
          </div>
        </section>

        <section>
          <h2>in utero SHS exposure modulated important biological functions and canonical pathways.</h2>
          <div style="display:grid;gap:15px">
            <p style="grid-column:1">
              Analyzed and visualized in Ingenuity pathway analysis (IPA).<br/><br/>
              All genes identified in the analysis were up-regulated.<br/><br/>
              Color intensities reflect SOM vs AOM.<br/><br/>
              Bars in brackets: <br/>
              Left: SOF vs AOF <br/>
              Right: SOM vs AOM.
            </p>
            <img style="grid-column:2/5;max-height:64vh" src="resource/SOFM_IPA.png" />
          </div>

          </section>
          <section>
          <h2>Potential epigenetic regulations by miRNAs</h2>
          <div style="display:grid;gap:15px">
            <p style="grid-row:1;grid-column:1">
            miRNAs are short noncoding RNAs that function as posttranscriptional modulators of gene expression by either <b>promoting mRNA degradation</b> or <b>blocking protein translation</b>.
            </p>
            <img style="grid-row:1/3;grid-column:2/5;max-height:55vh" src="resource/miRNA_mech.png" title="He L, Hannon GJ. 2004. MicroRNAs: small RNAs with a big role in gene regulation. Nat Rev Genet 5(7):522-531.
            " />
            <img style="grid-row:2;grid-column:1;max-height:15vh" src="resource/miRNATriangle1.png" class="#fragment" />
          </div>

          </section>

          <section>
          <h2>miRNA profiling revealed more miRNAs dysregulated by in utero SHS exposure than those biased by sex.</h2>
          <div style="display:grid;gap:15px;">
            <p style="grid-row:1;grid-column:1/3">GLM: ~ Smoke * Sex</p>
            <img style="grid-row:2;grid-column:1/3;max-height:36vh" src="resource/miRNA_venn.png" />
            <img style="grid-row:1/3;grid-column:3/6;max-height:64vh" src="resource/miRNA_heatmap.png" class="#fragment" />
          </div>
          </section>

          <section>
          <h2>In utero SHS exposure dysregulates miRNAs that are associated with lung inflammation and cancer.</h2>
          <p>
            ♀: SOF versus AOF, ♂: SOM versus AOM. N≥5 in each group for qRT-PCR. FDR&lt;0.05.
          </p>
          <table style="max-height:40vh;font-size:18px">
            <tr><th>miRNAs (mmu-)</th><th>Associated lung diseases & disorders</th><th>qRT-PCR Fold change</th></tr>
            <tr><td>miR-155-5p</td><td>Cancer (Tang et al. 2013; Yang et al. 2013); <br/>Lung inflammation (Tili et al. 2007);</td><td>♀: ↑2.4* ♂: ↑2.0*</td></tr>
            <tr><td>miR-541-5p</td><td></td><td>♀: ↓1.3 ♂: ↑1.8*</td></tr>
            <tr><td>miR-511-3p</td><td>Lung inflammation (Squadrito et al. 2012);<br/>Cancer (Zhang et al. 2012);</td><td>♀: ↑1.3 ♂: ↑1.7*</td></tr>
            <tr><td>miR-21-3p</td><td>Lung inflammation (Lu et al. 2009; Moschos et al. 2007); <br/>Cancer (Seike et al. 2009; Tang et al. 2013; Yang et al. 2013)</td><td>♀: ↑2.0 ♂: ↑2.4*
            <tr><td>miR-298-5p</td><td></td><td>♀: ↑2.0* ♂: ↑2.5*</td></tr>
            <tr><td>miR-18a-5p</td><td>Cancer (miR-17~92 cluster) (He et al. 2005; Hwang and Mendell 2007; Olive et al. 2010)</td><td>♀: ↑1.8* ♂: ↑1.6*
            <tr><td>miR-18b-5p</td><td></td><td>♀: ↑1.8 ♂: ↑3.6*</td></tr>
            <tr></tr>
          </table>
          </section>

          <section>
          <h2>Major cytogenetic bands were screened for transcriptional activity of both mRNA and miRNA.</h2>
          <p>
            Cytogenetic band information (114 in mouse) associated with genes and miRNAs is great resource to investigate the transcriptional activity over different chromosome locations.
          </p>
          <div style="display:grid;gap:15px;">
            <img style="grid-column:1;max-height:45vh;margin:auto" src="resource/miRNA_cytoband_basic.png" />
            <img style="grid-column:2;max-height:44vh;margin:auto" src="resource/miRNA_cytoband.png" class="#fragment" />
          </div>
          </section>

          <section>
          <h2>Inceased transcriptional activity points to a cluster of pro-inflammatory genes and miRNAs.</h2>
          <div style="display:grid;gap:15px;">
            <p style="grid-row:1;grid-column:1">
            Chr11C is enriched with upregulated genes and miRNAs.
            </p>
            <img style="grid-row:1/3;grid-column:2/5;max-height:75vh" src="resource/miRNA_cytoband_inflam.png" />
            <img style="grid-row:2;grid-column:1;max-height:15vh" src="resource/miRNATriangle2.png" class="#fragment" />
          </div>
          </section>

          <section>
          <h2>16 tumor suppressor genes are predicted targets and show inverse correlation with oncomir expression.</h2>
          <p style="font-size:68%">
            Spearman’s r ≤ -0.7, |Fold change| ≥ 1.5 <br/>
            <small>Zhao et al. 2013. TSGene: a web resource for tumor suppressor genes. Nucleic Acids Res. 41:D970-976.</small>
          </p>
          <div style="display:grid;gap:5px;grid-template-columns:1fr 1.5fr;">
            <div  style="grid-row:1;grid-column:1">
              <img style="max-height:48vh" src="resource/miRNA_db.png" />
            </div>
            <img style="grid-row:2;grid-column:1;max-height:15vh" src="resource/miRNATriangle3.png" class="#fragment" />
            <img style="grid-row:1/3;grid-column:2;max-height:75vh" src="resource/miRNA_anti_heatmap.png" class="#fragment" />
          </div>
        </section>
        <section>
          <h1>Summary</h1>
          <table style="font-size:50%">
            <tr>
              <th></th>
              <th>Integrated Omics</th>
              <th>Machine Learning</th>
              <th>External Database</th>
              <th>Workflow Automation</th>
            </tr>
            <tr>
              <th>LSU-PhD</th>
              <td>
                Phenome (lung);
                Transcrptome;
                Epigenome
              </td>
              <td></td>
              <td>Gene (WebGestalt);
                microRNA (miRBase, TargetScan)</td>
                <td>VBA macro; VB/WinForm app</td>
              </tr>
          </table>
        </section>
      </section>

      <section>
        <section>
          <h1>CU-PostDoc</h1>
          <hr style="margin:5%" />
          <h4>Lung injury and structural changes due to prolonged smoke exposure - Acute lung injury, Emphysema, ...</h4>
          <img style="max-width:24vw;max-height:18vh" src="resource/CU.png" />
        </section>

        <section>
          <h2>Areas of research at CU</h2>
          <p>
          The effects of adult chronic smoke exposure can be better examined by:
          <ol>
            <li>investigating microbiome to identify microbial organisms that are associated with disease progression and susceptibility differences.</li>
            <li>developing sensitive, unbiased, and robust computational methods for detecting changes in the lung.</li>
          </ol>
        </p>
          <ul>
            <li>Find suseptibity differences to smoke exposure across different strains of mice:</li>
            <ul>
              <li>Microbiome (16S rRNA-seq) – Phenome</li>
            </ul>
            <li>Extend capability of biomedical imaging</li>
            <ul>
              <li>Lung morphometry, automated, unbiased, sensitive - Early detection </li>
              <li>Tissue segmentation and cell categorization - Performance improvement </li>
            </ul>
          </ul>

          </section>
          <section>

          <h2>Hypothesis: Baseline microbiome profiles differentially modulate responses to cigarette smoke exposure</h2>
          <div style="display:grid;gap:5px;">
            <p style="grid-row:1;grid-column:1/5">
            Susceptibility to smoke induced lung injury differs across human population.
            <br/>
            Comparing differences across baseline vs response enables us to learn about microbiome predisposition.<br/>
            We look into the gut microbiome (abundant, well-characterized) across 4 strains of mice.
            </p>
            <img style="grid-row:2;grid-column:1;max-height:28vh;margin:auto" src="resource/mibiome_population.jpg" />
            <img style="grid-row:2;grid-column:2/5;max-height:36vh;margin:auto" src="resource/mibiome_4mice.png" />
          </div>


          </section>
          <section>

          <h2>Baseline strain-specific differences in the gut microbiome.</h2>
          <div style="display:grid;">
            <div style="grid-column:1/6">
            <img style="max-height:65vh" src="resource/mibiome_1circle.jpg" title= "Segata N, et al.
            GraPhlAn.
            " />
            </div>
            <div style="grid-column:6/10">
              <p>Most significant taxa tested in LEfSe (LDA>4.2):</p>
              <img style="max-height:42vh" src="resource/mibiome_baseline_4bar.jpg" title="Segata N, et al. Genome Biol. 2011. LEfSe (Linear discriminant analysis Effect Size).
              " />
            </div>
          </div>
          </section>

          <section>
          <h2>More inflammation and microbiome changes were found in AKR/J and C57BL/6J mice.</h2>
          <p>
          Beta diversity: variation of microbial communities, difference in taxonomic abundance profiles from different samples.
          <small>UniFrac: sequence distances (phylogenetic tree), based on the fraction of branch length shared between two samples or unique to one or the other sample.</small>
          </p>
          <div style="display:grid">
            <div style="grid-column:1">
              <img style="max-height:39vh;margin-bottom:0px" src="resource/mibiome_inflam.png" />
            </div>
            <div style="grid-column:2">
              <img style="max-height:39vh;margin-bottom:0px" src="resource/mibiome_PCoA.jpg" />
            </div>
          </div>
          </section>
          <section>

          <h2>Proinflammatory microbes induced by smoke exposure have high baseline in resistant strain.</h2>
          <p>
            <u>Hygiene hypothesis</u>: early exposure to particular microorganisms protects against allergic diseases by contributing to the development of the immune system. A lack of exposure is thought to lead to defects in the establishment of immune tolerance.
          </p>
          <div style="display:grid">
            <div style="grid-column:1">
              <p><small>
              <b>A/J</b>: Resistant strains with mild inflammation in the lung and blood.<br/>
              <b>AKR/J</b>: Susceptible strains with severe inflammation in the lung and blood.
              </small></p>
              <img style="max-height:33vh" src="resource/mibiome_1circle.jpg" />
            </div>
            <div style="grid-column:2">
              <img style="max-height:41vh" src="resource/mibiome_4circleA.png" />
            </div>
          </div>

          </section>
          <section>

          <h2>Lung Morphometric Analysis - Gridline Counting</h2>
          <p>Traditionally, people perform gridline counting to assess Mean Linear Intercept (MLI) to approximate the average airspace (µm). Drawbacks: labor intensive, time consuming.</p>
          <div style="display:grid;">
            <div style="grid-row:1/3;grid-column:1;margin:auto">
              <img style="max-height:26vh" src="resource/lung_healthy_des.jpg" />
            </div>
            <div style="grid-row:1;grid-column:2">
              <img style="max-height:22vh" src="resource/dp_emphy.jpg" />
            </div>
            <div style="grid-row:2;grid-column:2">
              <img style="max-height:22vh" src="resource/dp_healthy.jpg" />
            </div>
            <div style="grid-row:1/3;grid-column:3;margin:auto">
              <img style="max-height:26vh" src="resource/glc_basic.jpg" />
            </div>
            <div style="grid-row:3;grid-column:1/4">
            </div>
          </div>
          </section>

          <section>
            <h2>Computer vision and machine learning can be easily applied to extract airspaces and features.</h2>
            <div style="display:grid;grid-gap:15px">
              <div style="grid-row:1;grid-column:1">
                <img style="max-height:24vh" src="resource/dp_emphy_gc.jpg" />
              </div>
              <div style="grid-row:2;grid-column:1">
                <img style="max-height:24vh" src="resource/dp_emphy_cv.png" />
              </div>
              <div style="grid-row:1/3;grid-column:2">
                <p>Computer vision can instantly extract hundreds of enclosed airspaces and provide several important features for further analysis.<br/>
                  <u>Area</u> is the most <b>representative</b>, <b>robust</b> (not easily biased by scaling), <b>widely-supported</b> (across many software packages) feature.
                </p>
                <table style="font-size:65%;border: 1px solid black">
                  <tr>
                    <th>ID (#)​</th>
                    <th>Center of<br/>Gravity​</th>
                    <th>Width</th>
                    <th>Height</th>
                    <th style="color:darkred">Area​</th>
                    <th>...​</th>
                  </tr>
                  <tr>
                    <td>1​</td><td>...</td><td>...​</td><td>...​</td><td>...​</td><td>...​</td>
                  </tr>
                  <tr>
                    <td>2​</td><td>...</td><td>...​</td><td>...​</td><td>...​</td><td>...​</td>
                  </tr>
                  <tr>
                    <td>...​</td><td>...</td><td>...​</td><td>...​</td><td>...​</td><td>...​</td>
                  </tr>
                </table>
              </div>
            </div>
          </section>

          <section>
            <h2>Unweighted size distribution pointed to a subpopulation of small airspaces</h2>
            <div style="display:grid;gap:15px;grid-template-columns: 1.25fr 1fr;">
              <div style="grid-column:1">
                <p>
                  Small, red: µ=2.84, σ=0.49 <br/><br/>
                  $Area_{small} = 10^{2.84} = 631 \mu m^2$
                  $\begin{align}Diameter_{small} &=2*\sqrt{Area_{small}/\pi} \\
                  &=30µm
                  \end{align}$
                  <br/>
                  This is inline with the size of a single alveolus. <br/>
                  <b>"C57BL/6J mice (~35µm) ​ A/J strain (~38µm)"</b> <br/>
                  <i>Soutiere et al. 2004.​ <br/>
                  Differences in alveolar size in​ inbred mouse strains. <br/>
                  Respir Physiol Neurobiol 140(3):283-91.​</i>
                </p>
              </div>
              <div style="grid-column:2">
                <img style="max-height:50vh" src="resource/ml_1p.png" />
              </div>
            </div>
          </section>

          <section>
            <h2>Area-weighted size distribution identified two subpopulations of medium & large airspaces</h2>
            <div style="display:grid;gap:15px;grid-template-columns: 1.25fr 1fr;">
              <div style="grid-column:1">
                <p>
                  <span style="opacity:0.6">Small, dotted: µ=2.84, σ=0.49 </span><br/>
                  Medium, red: µ=3.38 σ=0.48​ <br/>
                  Large, green: µ=4.34 σ=0.30 <br/>
                  <br/>
                  $Area_{Medium}=10^{3.38}=2399\mu m2​$  (4X) <br/>
                  $Diameter_{Medium}=56\mu m​$  (2X) <br/>
                  <br/>
                  $Area_{Large}=10^{4.34}=21878\mu m2​$  (35X) <br/>
                  $Diameter_{Large}=167\mu m​$  (6X) <br/>
                  <br/>
                  These sizes are inline with the <b>alveolar sac</b> and <b>alveolar duct</b> / <b>area of destruction</b>. <br/>
                </p>
              </div>
              <div style="grid-column:2">
                <img style="max-height:50vh" src="resource/ml_2p.png" />
              </div>
            </div>
            <p>Colored as <span style="color:green;font-weight:900;-webkit-text-stroke: .8px black">alveolus</span>, <span style="color:yellow;font-weight:900;-webkit-text-stroke: .8px black">alveolar sac</span>, and <span style="color:red;font-weight:900;-webkit-text-stroke: .8px black">alveolar duct & area of destruction</span>.
            </p>
          </section>

          <section>
            <h2>Ductal/Destructive Fraction (DF) is a more sensitive and robust indicator than MLI.</h2>
            <div style="display:grid;grid-gap:0px;" >
              <p style="grid-row:1;grid-column:1/3;margin:5px">Healthy control<br/> MLI=62µm DF=23%</p>
              <img style="grid-row:2;grid-column:1;max-height:20vh;margin:0px" src="resource/dp_healthy_gc.jpg" />
              <img style="grid-row:2;grid-column:2;max-height:20vh;margin:0px" src="resource/dp_healthy_pap.jpg" />
              <p style="grid-row:1;grid-column:3/5;margin:5px">Emphysema<br/> MLI=83µm <b>(↑34%)</b> DF=44% <b>(↑91%)</b>​</p>
              <img style="grid-row:2;grid-column:3;max-height:20vh;margin:0px" src="resource/dp_emphy_gc.jpg" />
              <img style="grid-row:2;grid-column:4;max-height:20vh;margin:0px" src="resource/dp_emphy_pap.jpg" />
              <img style="grid-row:3;grid-column:1/4;max-height:24vh;margin:15px auto" src="resource/pap_robust.jpg" />
              <img style="grid-row:3;grid-column:4;max-height:24vh;margin:15px auto" src="resource/ajrcmb.gif" />
            </div>
          </section>


        <section>
          <h2>Prototyped in R, further developed into a C#/WPF desktop-app for better speed and user experience.</h2>
          <div style="display:grid;grid-gap:10px" >
            <div style="grid-row:1/3;grid-column:1" >
              <p style="margin:0px;text-align:center;" >
                ~1000 lines of XAML code for UI​
              </p>
              <img style="max-height:26vh;margin:auto" src="resource/pap_dev1.png" />
            </div>
            <div style="grid-row:1/3;grid-column:2">
            <p style="margin:0px;text-align:center" >
              ~3000 lines of C# code for computation​
            </p>
            <img style="max-height:26vh;margin:auto" src="resource/pap_dev2.png" />
            </div>
          <img class="#fragment" style="grid-row:2/4;grid-column:1/3;max-height:38vh;margin:auto" src="resource/pap_screenshot.png" />
        </section>

        <section>
          <h2>Convolutional neural network is capable of tissue segmentation and cell classification.</h2>
          <div style="display:grid;grid-gap:5px" >
            <p style="grid-row:1;grid-column:3;margin:0px">
              CNN applies a sliding windows to first extract low-level features and layer-by-layer captures high-level features enough to perform <b>classification</b>, <b>segmentation</b>, and <b>instance/object detection</b> tasks. <br/><br/>
              semantic <--> uncountable tissue <br/>
              instance <--> countable cells
            </p>
            <img style="grid-row:1;grid-column:1/3;max-height:34vh;margin:0px" src="resource/cnn_basic.png" />
            <img style="grid-row:2;grid-column:1/4;max-height:18vh;margin:0px" src="resource/segmentation_types.jpg" title="https://stackoverflow.com/questions/33947823/what-is-semantic-segmentation-compared-to-segmentation-and-scene-labeling"/>
        </div>
        </section>

      <section>
        <div>
          <h2>Precise segmentation can be achieved and optimized based on UNet architecture.</h2>
          <div style="display:grid;grid-gap:0px;grid-template-rows: 1.8fr 1.3fr 0.4fr;font-size:84%">
            <img style="grid-row:1/3;grid-column:1/6;margin:0px auto;max-height:37vh;" src="resource/cnn_unet.jpg" />
            <p style="grid-row:1;grid-column:2/4;text-align:center">
              Training input is the same while the B&W training output can be the background, two normal histological regions of the lung, or pathological lesions.
            </p>
            <img style="grid-row:2/4;grid-column:2/4;max-height:29vh;margin:auto" src="resource/dp_6seg.jpg" />
            <p style="grid-row:3;grid-column:1">CNN backbone optimization:​ VGG16 -> ResNet -> DenseNet​</p>
            <p style="grid-row:3;grid-column:4">Six-Color Segmentation Map​: Multi-Class Accuracy 96%​</p>
          </div>
        </div>
      </section>

      <section>
        <h2>Cell classification, counting and outlining can be performed via Mask R-CNN</h2>
        <div style="display:grid;grid-gap:0px" >
          <!-- <img src="resource/cnn_faster_rcnn_eg.png" title="https://medium.com/@jonathan_hui/image-segmentation-with-mask-r-cnn-ebe6d793272​" /> -->
          <img style="grid-row:1;grid-column:1;margin:-10px 0px;max-height:24vh" src="resource/cnn_mask_rcnn_eg.png" title="https://medium.com/@jonathan_hui/image-segmentation-with-mask-r-cnn-ebe6d793272​" />
          <img style="grid-row:1;grid-column:2;margin:-10px 0px;max-height:33vh" src="resource/cnn_mask_rcnn.png" />
          <div style="grid-row:2;grid-column:1/3;max-height:30vh;overflow: hidden;margin:-20px auto">
            <img style="max-height:45vh;object-fit:cover;object-position:top;" src="resource/cnn_mask_rcnn_applied.jpg" />
          </div>
        </div>
      </section>
      <section>
        <h1>Summary</h1>
        <table style="font-size:50%">
          <tr>
            <th></th>
            <th>Integrated Omics</th>
            <th>Machine Learning</th>
            <th>External Database</th>
            <th>Workflow Automation</th>
          </tr>
          <tr>
            <th>LSU-PhD</th>
            <td>
              Phenome (lung);
              Transcrptome;
              Epigenome
            </td>
            <td></td>
            <td>Gene (WebGestalt);
              microRNA (miRBase, TargetScan)</td>
              <td>VBA macro; VB/WinForm app</td>
            </tr>
          <tr>
            <th>CU-PostDoc</th>
            <td>
              Microbiome (gut, lung)<br/>
            </td>
            <td>
              CV (airspace feature);
              GMM (airspace categorization);
              DL/CNN (tissue seg, cell recog)
            </td>
            <td></td>
            <td>
              R script for gridline counting;
              C#/WPF app for parenchymal airspace profiling
            </td>
          </tr>
        </table>
      </section>

        </section>

            <section>
                <section>
                    <h1>Asuragen</h1>
          <hr style="margin:5%" />
          <h4>Unified and accurate genetic variant calling and risk assessment pipeline - Next-gen & 3rd-gen sequencing,
            bioinfo pipeline engineering and algorithm development.</h4>
                    <img style="max-width:24vw;max-height:18vh" src="resource/Asuragen.png" />
                </section>
                <section>
                  <h1>Summary</h1>
                  <table style="font-size:50%">
                    <tr>
                      <th></th>
                      <th>Integrated Omics</th>
                      <th>Machine Learning</th>
                      <th>External Database</th>
                      <th>Workflow Automation</th>
                    </tr>
                    <tr>
                      <th>LSU-PhD</th>
                      <td>
                        Phenome (lung);
                        Transcrptome;
                        Epigenome
                      </td>
                      <td></td>
                      <td>Gene (WebGestalt);
                        microRNA (miRBase, TargetScan)</td>
                        <td>VBA macro; VB/WinForm app</td>
                      </tr>
                    <tr>
                      <th>CU-PostDoc</th>
                      <td>
                        Microbiome (gut, lung)<br/>
                      </td>
                      <td>
                        CV (airspace feature);
                        GMM (airspace categorization);
                        DL/CNN (tissue seg, cell recog)
                      </td>
                      <td></td>
                      <td>
                        R script for gridline counting;
                        C#/WPF app for parenchymal airspace profiling
                      </td>
                    </tr>
                    <tr>
                      <th>Asuragen</th>
                      <td>
                        Genome (human)<br/>
                      </td>
                      <td>
                        DL (variant calling)
                      </td>
                      <td>
                        gnomAD; ClinVar
                      </td>
                      <td>Unified bioinfo pipeline built with Nextflow</td>
                    </tr>
                  </table>
                </section>
            </section>


            <section>
                <section>
                    <h1>MyoKardia</h1>
          <hr style="margin:5%" />
                    <h4>Omics profiling across different compound treatments and animal models of cardiovascular diseases.</h4>
                    <img style="max-width:24vw;max-height:18vh" src="resource/Myokardia.png" />
                </section>
                <section>
                  <h1>Summary</h1>
                  <table style="font-size:50%">
                    <tr>
                      <th></th>
                      <th>Integrated Omics</th>
                      <th>Machine Learning</th>
                      <th>External Database</th>
                      <th>Workflow Automation</th>
                    </tr>
                    <tr>
                      <th>LSU-PhD</th>
                      <td>
                        Phenome (lung);
                        Transcrptome;
                        Epigenome
                      </td>
                      <td></td>
                      <td>Gene (WebGestalt);
                        microRNA (miRBase, TargetScan)</td>
                        <td>VBA macro; VB/WinForm app</td>
                      </tr>
                    <tr>
                      <th>CU-PostDoc</th>
                      <td>
                        Microbiome (gut, lung)<br/>
                      </td>
                      <td>
                        CV (airspace feature);
                        GMM (airspace categorization);
                        DL/CNN (tissue seg, cell recog)
                      </td>
                      <td></td>
                      <td>
                        R script for gridline counting;
                        C#/WPF app for parenchymal airspace profiling
                      </td>
                    </tr>
                    <tr>
                      <th>Asuragen</th>
                      <td>
                        Genome (human)<br/>
                      </td>
                      <td>
                        DL (variant calling)
                      </td>
                      <td>
                        gnomAD; ClinVar
                      </td>
                      <td>Unified bioinfo pipeline built with Nextflow</td>
                    </tr>
                    <tr>
                      <th>MyoKardia</th>
                      <td>
                        Phenome (heart);
                        Transcriptome
                      </td>
                      <td></td>
                      <td>Gene (clusterProfiler)</td>
                      <td>R/Shiny app</td>
                    </tr>
                  </table>
                </section>
            </section>

            <section>
              <section>
                <h1>BMS</h1>
                <hr style="margin:5%" />
                <h4>Diverse exposure to target nomination, novel therapeutic modality, consortium datamining, ...</h4>
                <img style="max-width:24vw;max-height:18vh" src="resource/BMS.png" />
              </section>
              <section>
                <h1>Summary</h1>
                <table style="font-size:50%">
                  <tr>
                    <th></th>
                    <th>Integrated Omics</th>
                    <th>Machine Learning</th>
                    <th>External Database</th>
                    <th>Workflow Automation</th>
                  </tr>
                  <tr>
                    <th>LSU-PhD</th>
                    <td>
                      Phenome (lung);
                      Transcrptome;
                      Epigenome
                    </td>
                    <td></td>
                    <td>Gene (WebGestalt);
                      microRNA (miRBase, TargetScan)</td>
                      <td>VBA macro; VB/WinForm app</td>
                    </tr>
                  <tr>
                    <th>CU-PostDoc</th>
                    <td>
                      Microbiome (gut, lung)<br/>
                    </td>
                    <td>
                      CV (airspace feature);
                      GMM (airspace categorization);
                      DL/CNN (tissue seg, cell recog)
                    </td>
                    <td></td>
                    <td>
                      R script for gridline counting;
                      C#/WPF app for parenchymal airspace profiling
                    </td>
                  </tr>
                  <tr>
                    <th>Asuragen</th>
                    <td>
                      Genome (human)<br/>
                    </td>
                    <td>
                      DL (variant calling)
                    </td>
                    <td>
                      gnomAD; ClinVar
                    </td>
                    <td>Unified bioinfo pipeline built with Nextflow</td>
                  </tr>
                  <tr>
                    <th>MyoKardia</th>
                    <td>
                      Phenome (heart);
                      Transcriptome
                    </td>
                    <td></td>
                    <td>Gene (clusterProfiler)</td>
                    <td>R/Shiny app</td>
                  </tr>
                  <tr>
                    <th>BMS</th>
                    <td>
                      Translatome as in Ribo-seq
                    </td>
                    <td>
                      NLP (medical records);
                      Ranking (targets);
                      Feature importance (biomarkers);
                      VAE for AAV capsid evo.
                    </td>
                    <td>BioBank, Heart Failure Consortium</td>
                    <td>RMarkdown-based analysis & visualization toolkit</td>
                  </tr>
                  <tr>
                    <th></th>
                    <th>In-depth profiling to study omics interactions and disease mechanism.</th>
                    <th>Leverage AI for improved assessment and optimized path.</th>
                    <th>Mine dataset for target ideation or put studies in a broder context.</th>
                    <th>Work efficiently and enpower the entire team.</th>
                  </tr>
                </table>
              </section>

            </section>
        </div>
    </div>

    <script src="lib/js/head.min.js"></script>
    <script src="js/reveal.js"></script>
    <script>
        Reveal.initialize({
            // reveal.js configuration, cf. https://github.com/hakimel/reveal.js/#configuration
            controls: true,
            progress: true,
            history: true,
            center: true,
            zoom: true,
            // transition: 'convex',
            transition: 'page',
            slideNumber: 'c/t',
            margin: 0.05,
      // width: 960, height: 800,
      width: 1100, height: 800,
            // width: 1920, height: 1080,

            // Optional libraries used to extend on reveal.js
            dependencies: [
                { src: 'lib/js/classList.js', condition: function () { return !document.body.classList; } },

                // enable built-in plugins
                { src: 'plugin/markdown/marked.js', condition: function () { return !!document.querySelector('[data-markdown]'); } },
                { src: 'plugin/markdown/markdown.js', condition: function () { return !!document.querySelector('[data-markdown]'); } },
                { src: 'plugin/highlight/highlight.js', async: true, callback: function () { hljs.initHighlightingOnLoad(); } },
                { src: 'plugin/zoom/zoom.js' },
                { src: 'plugin/notes/notes.js' },
                { src: 'plugin/math/math.js' },

                // enable additionally installed plugins
                { src: 'plugin/menu/menu.js' },
                { src: 'plugin/title-footer/title-footer.js', async: true, callback: function () { title_footer.initialize(); } },
                { src: 'plugin/toc-progress/toc-progress.js', async: true, callback: function () { toc_progress.initialize(); toc_progress.create(); } },
            ],

            // plugin settings

            // plugin math, cf. https://github.com/hakimel/reveal.js#mathjax
            math: {
                mathjax: 'js/MathJax.js',
                config: 'TeX-AMS_HTML-full'  // See http://docs.mathjax.org/en/latest/config-files.html
            },

            // plugin menu, cf. https://github.com/denehyg/reveal.js-menu#configuration
            menu: {
                side: 'left',
                numbers: false,
                titleSelector: 'h1, h2, h3, h4, h5, h6',
                hideMissingTitles: false,
                markers: false,
                custom: false,
                themes: [
                    { name: 'Black', theme: 'css/theme/black.css' },
                    { name: 'White', theme: 'css/theme/white.css' },
                    { name: 'League', theme: 'css/theme/league.css' },
                    { name: 'Sky', theme: 'css/theme/sky.css' },
                    { name: 'Beige', theme: 'css/theme/beige.css' },
                    { name: 'Simple', theme: 'css/theme/simple.css' },
                    { name: 'Serif', theme: 'css/theme/serif.css' },
                    { name: 'Blood', theme: 'css/theme/blood.css' },
                    { name: 'Night', theme: 'css/theme/night.css' },
                    { name: 'Moon', theme: 'css/theme/moon.css' },
                    { name: 'Solarized', theme: 'css/theme/solarized.css' }
                ],
                transitions: true,
                openButton: true,
                openSlideNumber: true,
                keyboard: true
            },
        });
    </script>
</body>

</html>


<!--
file created by github.com/theno/fabsetup
substituted variables:
{{presi_title_space}}: reveal.js template
-->